usaspending_api/references/models.py

```diff
@@ -383,6 +383,14 @@ class LegalEntity(DataSourceTrackedModel):
     small_business_description = models.TextField(blank=True, null=True)
     individual = models.CharField(max_length=1, blank=True, null=True)
 
+    def save(self, *args, **kwargs):
+        super(LegalEntity, self).save(*args, **kwargs)
+
+        try:
```

Comment:
Is there a way that Django's `get_or_create` pattern would work here? Overall, that seems like a cleaner way than try/catching db errors?